Android compile error: ReactNativeContacts is not abstract and does not override abstract method createJSModules() in ReactPackage

Since most recent RN-versions (0.50 here) the following compilation error occurs with android:

> ReactNativeContacts is not abstract and does not override abstract method createJSModules() in ReactPackage

```
./node_modules/react-native-contacts/android/src/main/java/com/rt2zz/reactnativecontacts/ReactNativeContacts.java:13: error: ReactNativeContacts is not abstract and does not override abstract method createJSModules() in ReactPackage
public class ReactNativeContacts implements ReactPackage {
       ^
1 error
```

--- a/android/src/main/java/com/rt2zz/reactnativecontacts/ReactNativeContacts.java
+++ b/android/src/main/java/com/rt2zz/reactnativecontacts/ReactNativeContacts.java
@@ -5,9 +5,11 @@ import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.uimanager.ViewManager;
+import com.facebook.react.bridge.JavaScriptModule;
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 public class ReactNativeContacts implements ReactPackage {
@@ -19,6 +21,11 @@ public class ReactNativeContacts implements ReactPackage {
 
         modules.add(new ContactsManager(reactContext));
         return modules;
+    }
+
+    @Override 
+    public List<Class<? extends JavaScriptModule>> createJSModules() { 
+        return Collections.emptyList(); 
     }
 
     @Override

